platform-config/blogStarter/analytics/dashboard/blogDashboard.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can we move the entities vs config into separate folders per this thread? https://yext.slack.com/archives/C05KC4UFCUW/p1695392605769319